src/test/ui/emit-directives.stderr

```diff
@@ -0,0 +1 @@
+{"directive":"metadata file written: $TEST_BUILD_DIR/emit-directives/a"}
```

Comment:
I'm curious why this ends with `/a`? I would have expected something like `libemit-directives.rmeta`. Should the crate have `#![crate_type = "rlib"]` to possibly rename it?

Reply:
The output file for compiletests is always called `a`.

Reply:
Yeah, I just used what the test harness produced :)